Source/OrchBase/AppStructures/EdgeI_t.cpp

```diff
@@ -35,8 +35,6 @@ fprintf(fp,"%sNameBase       %s\n",os,FullName().c_str());
 fprintf(fp,"%sMe,Parent      0x%#018lx,0x%#018lx\n",
             os,(uint64_t)this,(uint64_t)par);
 if (par!=0) fprintf(fp,"%s...%s\n",os,par->FullName().c_str());
-fprintf(fp,"%sP-address :\n",os);
```

Comment:
Should this not be replaced with (as much as possible of) the full symbolic address? 🍠

Reply:
As far as I can tell, it's not needed. The Composer dictates what the software address corresponding to this edge is, and the composer is the only part of the Orchestrator that cares.

Reply:
It dictates what the PinAddr is, but not the rest of the software address. If this is being printed below a device dump, then all will be fine. If this is being printed in isolation, then something to say where the edge belongs might be needed?

Reply:
I have no idea where to go in the rabbit hole to find this out :p

Reply:
As good a reason as any. Ship it!